tests/multi_hist_diff_expected.json

```diff
@@ -0,0 +1,73 @@
+[
```

Comment:
done, and fixed the pip differ and tests so that the path is only what is necessary.